tests/test_pump.py

```diff
@@ -26,8 +26,8 @@ def test_pump_4_3_liters(self):
         # verify that the values are equal to several significant digits.
         self.assertAlmostEqual(wait_time_expected, wait_time_actual)
         # Verify the signal gets sent to turn the pump pin on, then off.
-        self.mock_pi_io.turn_pin_on.assert_called_once_with(pi_io.PIN_PUMP)
-        self.mock_pi_io.turn_pin_off.assert_called_once_with(pi_io.PIN_PUMP)
+        self.mock_pi_io.turn_pin_on.assert_called_once_with(self.pump_pin)
+        self.mock_pi_io.turn_pin_off.assert_called_once_with(self.pump_pin)
```

Comment:
(optional) Consider using [`assert_has_calls`](https://docs.python.org/dev/library/unittest.mock.html#unittest.mock.Mock.assert_has_calls) because it enforces an ordering to the calls. But if it's a pain to get the syntax right, I think the current assertions are fine.